src/BottomComponent.css

```diff
@@ -0,0 +1,31 @@
+.bottom {
+    background: #0202027d;
+    max-height: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.bottom--left {
+    grid-area: leftBottom;
+}
+
+.bottom.bottom--right {
+    grid-area: rightBottom;
+}
+
+.play-btn {
+    flex: 1;
+}
+
+.play-btn > button {
+    border-radius: 50%;
+    width: 80px;
+    height: 80px;
+    border: 3px solid darkgray;
+    background: #2b2b2b;
+}
+
+svg {
```

Comment:
oj nie można takich selektorów robić, ponieważ są zbyt ogólnie. Pomimo, że tworzysz swoje style koło kompentów nie oznacza, że one działają jeydnie w ich konkteście. To jest nadal jeden wielki css tylko, że łatwiej nam tym kodem zarządzać

Reply:
done